src/main/java/gregtech/integration/jei/multiblock/MultiblockInfoCategory.java

```diff
@@ -84,4 +88,10 @@ public void setRecipe(IRecipeLayout recipeLayout, MultiblockInfoRecipeWrapper re
         recipeWrapper.setRecipeLayout((RecipeLayout) recipeLayout, this.guiHelper);
     }
 
+    @Override
+    public void drawExtras(Minecraft minecraft) {
+        for (int i = 0; i < 20; ++i) {
+            this.slot.draw(minecraft, 18*i-180*(i/10), background.getHeight()-36+18*(i/10));
+        }
+    }
```

Comment:
I don't think this should be extracted here. As it does not bring us any advantage. Also if we need to change number of rendered slots we would need to do it in two files

Reply:
This was fixed when the rendering fix/cleanup was merged in